pxtblocks/blocklylayout.ts

```diff
@@ -298,6 +298,8 @@ namespace pxt.blocks.layout {
             .then((customCss) => {
                 const blocklySvg = Util.toArray(document.head.querySelectorAll("style"))
                     .filter((el: HTMLStyleElement) => /\.blocklySvg/.test(el.innerText))[0] as HTMLStyleElement;
+                // Custom CSS injected directly into the DOM by Blockly
+                customCss.push((document.getElementById(`blockly-renderer-style-pxt`) as HTMLLinkElement)?.innerText);
```

Comment:
```suggestion
                customCss.push((document.getElementById(`blockly-renderer-style-pxt`) as HTMLLinkElement)?.innerText ?? "");
```

to avoid adding in 'undefined' to the cssString if the element doesn't exist

Reply:
thanks, bundled in with the other quick tweak moving this sheet to the front of the array to match precedence